ui/desktop/app/routes/scopes/scope/projects/sessions.js

```diff
@@ -83,5 +84,6 @@ export default class ScopesScopeProjectsSessionsRoute extends Route {
   @notifySuccess('notifications.canceled-success')
   async cancelSession(session) {
     await session.cancelSession();
+    await this.ipc.invoke('cancel', { session_id: session.id });
```

Comment:
On the IPC side, it's more of a "stop" or "kill" concept if the session has already been canceled via the API.

Reply:
Updated to 'stop'. Ty!